srcpkgs/python-CodeIntel/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'python-CodeIntel'
+pkgname=python-CodeIntel
+version=2.0.0
+revision=1
+noarch=yes
```

Comment:
It is shipping with `*.so` so files so it is architecture dependent. remove noarch.

Reply:
Okay did remove it. Thanks